tensorflow/python/data/kernel_tests/iterator_test.py

```diff
@@ -778,34 +799,43 @@ def testRepeatedGetNextWarning(self):
           iterator_ops.GET_NEXT_CALL_WARNING_MESSAGE, str(warning.message))
 
   # pylint: disable=g-long-lambda
-  @parameterized.named_parameters(
-      ("Tensor", lambda: constant_op.constant(37.0),
-       tensor_spec.TensorSpec([],
-                              dtypes.float32), ops.Tensor, dtypes.float32, []),
-      ("SparseTensor", lambda: sparse_tensor.SparseTensor(
-          indices=[[0]],
-          values=constant_op.constant([0], dtype=dtypes.int32),
-          dense_shape=[1]), sparse_tensor.SparseTensorSpec([1], dtypes.int32),
-       sparse_tensor.SparseTensor, dtypes.int32, [1]),
-      ("Nest", lambda: {
-          "a": constant_op.constant(37.0),
-          "b": (constant_op.constant(["Foo"]), constant_op.constant("Bar"))
-      }, {
-          "a":
-              tensor_spec.TensorSpec([], dtypes.float32),
-          "b": (tensor_spec.TensorSpec(
-              [1], dtypes.string), tensor_spec.TensorSpec([], dtypes.string))
-      }, {
-          "a": ops.Tensor,
-          "b": (ops.Tensor, ops.Tensor)
-      }, {
-          "a": dtypes.float32,
-          "b": (dtypes.string, dtypes.string)
-      }, {
-          "a": [],
-          "b": ([1], [])
-      }),
-  )
+  @combinations.generate(combinations.times(
```

Comment:
As a rule of thumb, we have agreed on the tf.data team that going forward we should not used parametrized tests that require `lambda` parameters and instead create separate unit tests. Could you please split this tests into separate tests, one for each test case? Thank you.

Reply:
Sure, will do it.